src/engine/strat_engine/pool.rs

```diff
@@ -117,6 +117,19 @@ impl StratPool {
     pub fn has_filesystems(&self) -> bool {
         self.thin_pool.has_filesystems()
     }
+
+    /// The names of DM devices belonging to this pool that may generate events
+    pub fn get_eventing_dev_names(&self) -> Vec<DmNameBuf> {
+        self.thin_pool.get_eventing_dev_names()
+    }
+
+    pub fn event_on(&mut self, _dm_name: &DmName) -> EngineResult<()> {
+        // We could be smart and just check the thing that evented, but for
```

Comment:
Could you turn this into a ```TODO``` or a ```FIXME```, please?

Reply:
ok